src/main/java/duke/Deadline.java

```diff
@@ -0,0 +1,35 @@
+package duke;
+
+/**
+ * Represent a deadline task
+ * It corresponds to a deadline represent by a string of task and the time
+ */
+public class Deadline extends Task{
+
+    /**
+     * Construtor of deadline
```

Comment:
Is there a typo here?
```suggestion
     * Constructor of deadline
```